resources/js/components/persistence/persistence.ts

```diff
@@ -42,19 +49,19 @@ function deserialise<T, S>(serialiser: Serialiser<T, S>, storedSerialisedValue:
   }
 }
 
-/**
- * Persists a value in the local storage.
- */
+/** Persists a value in the local storage. */
 export function createLocalStoragePersistence<T>({
```

Comment:
Może dać <T=unknown>, żeby wymusić napisanie typu explicite (inaczej może być inferowany)?

Reply:
Próbowałem taki efekt uzyskać żeby było trzeba podać, ale nie da się, takie z domyślnym też inferuje, i tak w sumie bym obstawiał że jest. Ale gdy ten typ jest dostatecznie skomplikowany to i tak trzeba go podać bo się nie inferuje w pełni, więc w tym nadzieja.

Zresztą rzeczy persistowane mniej trwale, na przykład w URLu, nie będą chyba potrzebowały aż tylu zabezpieczeń.